### ISSUE TITLE
Cannot install homebrew - -e:280:in `<main>': invalid byte sequence in US-ASCII (ArgumentError)

### ISSUE THREAD

**Issue by justechn** [Homebrew/brew#2831]
Hey guys, I am trying to install homebrew from the instructions found on https://brew.sh/ but I get an error ```-e:280:in `<main>': invalid byte sequence in US-ASCII (ArgumentError)```

The command I typed is `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"` and the output from that command is below

```
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew
==> The following new directories will be created:
/usr/local/Cellar
/usr/local/Homebrew
/usr/local/Frameworks
/usr/local/etc
/usr/local/opt
/usr/local/sbin
/usr/local/share/zsh
/usr/local/share/zsh/site-functions
/usr/local/var

Press RETURN to continue or any other key to abort
==> /usr/bin/sudo /bin/mkdir -p /usr/local/Cellar /usr/local/Homebrew /usr/local/Frameworks /usr/local/etc /usr/local/opt /usr/local/sbin /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var
Password:
==> /usr/bin/sudo /bin/chmod g+rwx /usr/local/Cellar /usr/local/Homebrew /usr/local/Frameworks /usr/local/etc /usr/local/opt /usr/local/sbin /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var
==> /usr/bin/sudo /bin/chmod 755 /usr/local/share/zsh /usr/local/share/zsh/site-functions
==> /usr/bin/sudo /usr/sbin/chown admin /usr/local/Cellar /usr/local/Homebrew /usr/local/Frameworks /usr/local/etc /usr/local/opt /usr/local/sbin /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var
==> /usr/bin/sudo /usr/bin/chgrp admin /usr/local/Cellar /usr/local/Homebrew /usr/local/Frameworks /usr/local/etc /usr/local/opt /usr/local/sbin /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var
==> /usr/bin/sudo /bin/mkdir -p /Users/admin/Library/Caches/Homebrew
==> /usr/bin/sudo /bin/chmod g+rwx /Users/admin/Library/Caches/Homebrew
==> /usr/bin/sudo /usr/sbin/chown admin /Users/admin/Library/Caches/Homebrew
==> /usr/bin/sudo /bin/mkdir -p /Library/Caches/Homebrew
==> /usr/bin/sudo /bin/chmod g+rwx /Library/Caches/Homebrew
==> /usr/bin/sudo /usr/sbin/chown admin /Library/Caches/Homebrew
-e:280:in `<main>': invalid byte sequence in US-ASCII (ArgumentError)
```

I tried running the command again, but I get the same error, just this time without all the output

Any help would be appreciated.

**Comment by MikeMcQuaid** [comment]
Sorry, not sure what's going on here. You may want to follow an alternate install here: http://docs.brew.sh/Installation.html

**Comment by justechn** [comment]
really? that's it? no request for additional information, no attempt to debug, just a frank dismissal? that is some great service you offer here.

**Comment by ilovezfs** [comment]
https://github.com/Homebrew/legacy-homebrew/issues/41632
https://github.com/Homebrew/legacy-homebrew/issues/44744

So it may be an unusual locale setting you have?

**Comment by MikeMcQuaid** [comment]
> really? that's it? no request for additional information, no attempt to debug, just a frank dismissal? that is some great service you offer here.

We are an understaffed, underfunded project run by volunteers mostly in our spare time. You asked us for help with an issue affecting just you (and didn't fill in the issue template, as requested, or submit this to the proper repository) and offered a workaround and closed it. In response you are sarcastic, rude and insulting towards this project. Do you think this makes us more or less likely to want to either work on this project or help you in future?

Finally, I quote our license:
> THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED.

In short: if it doesn't work: that's your problem, not ours. This is the same for all open source software and our support obligations extend as far as the amount you paid for the software: nothing. Thankfully: you have access to the source code so can fix it yourself.

**Comment by rejsmont** [comment]
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```
